demo/editor.html

```diff
@@ -1,4 +1,9 @@
 <!doctype html>
+<!--
+title: Grid Editor
```

Comment:
This should not be listed in the docs site, because it’s not available. Could be removed completely from the repo.